Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -177,6 +190,14 @@ public final class ProjectPamphletViewController: UIViewController {
     }
   }
 
+  private func goToDeprecatedRewards(project: Project, refTag: RefTag?) {
```

Comment:
The naming of this function is a little confusing because really both go to rewards, just one is presented using a deprecated presentation style (ie. no custom transition), right?